lib/turn.rb

```diff
@@ -0,0 +1,24 @@
+class Turn
+  attr_reader :guess, :card
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    if guess == card.answer
+      true
+    else
+      false
+    end
+  end
+
+  def feedback
+    if correct? == true
```

Comment:
since `correct?` will return true or false, you do not need to have the `==true` on this line.